.github/workflows/upgrade_downgrade_test_backups_e2e.yml

```diff
@@ -62,20 +62,14 @@ jobs:
             - 'bootstrap.sh'
             - '.github/workflows/upgrade_downgrade_test_backups_e2e.yml'
 
-    - name: Set up Go
+    - name: Tune the OS
       if: steps.changes.outputs.end_to_end == 'true'
-      uses: actions/setup-go@d35c59abb061a4a6fb18e82ac0862c26744d6ab5 # v5.5.0
-      with:
-        go-version: 1.24.7
```

Comment:
Seems like we missed upgrading this value when v23 was released. Changing this to not hardcode this value will ensure we won't miss updating this value in the future.